Done! Two questions / confusions about the test suite, which I can create issues for if helpful. Broadly, is there some special way you are running the tests or just `python -m unittest`?

1. I changed **one** test (in test_pandas_dataset.py) so it would fail, and ran the full test suite using `python -m unittest`. The output showed me that there were **two** failures. However, running only the tests in test_pandas_dataset.py showed just one failure. Do you know why this might be?
2. The failure message was fairly unhelpful, eg:
```
======================================================================
FAIL: test_expect_column_to_exist (tests.test_pandas_dataset.TestPandasDataset)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/louispotok/great_expectations/tests/test_pandas_dataset.py", line 28, in test_expect_column_to_exist
    self.assertEqual(out, t['out'])
AssertionError: {'success': False} != {'success': True}
- {'success': False}
?             ^^^^

+ {'success': True}
?             ^^^

```
Notice this does not show me *which* test case failed. I think this is due to the test style of `for c in cases: assertEquals(f(c["in"]), c["out"])`. One way to fix this would be to pass c["notes"] to `assertEquals` every time, but this is just more boilerplate. Have you encountered this, or do you have other thoughts?

--- a/great_expectations/dataset/base.py
+++ b/great_expectations/dataset/base.py
@@ -852,7 +852,7 @@ If you wish to change this behavior, please set discard_failed_expectations, dis
     ##### Table shape expectations #####
 
     def expect_column_to_exist(
-            self, column, column_idx=None, result_format=None, include_config=False, 
+            self, column, column_index=None, result_format=None, include_config=False, 
             catch_exceptions=None, meta=None
         ):
         """Expect the specified column to exist.
@@ -865,9 +865,9 @@ If you wish to change this behavior, please set discard_failed_expectations, dis
                 The column name.
 
         Other Parameters:
-            column_idx (int or None): \
+            column_index (int or None): \
                 If not None, checks the order of the columns. The expectation will fail if the \
-                column is not in location column_idx.
+                column is not in location column_index.
             result_format (str or None): \
                 Which output mode to use: `BOOLEAN_ONLY`, `BASIC`, `COMPLETE`, or `SUMMARY`.
                 For more detail, see :ref:`result_format <result_format>`.

--- a/great_expectations/dataset/pandas_dataset.py
+++ b/great_expectations/dataset/pandas_dataset.py
@@ -174,13 +174,13 @@ class PandasDataSet(MetaPandasDataSet, pd.DataFrame):
     @DocInherit
     @DataSet.expectation(['column'])
     def expect_column_to_exist(
-            self, column, column_idx=None, result_format=None, include_config=False, 
+            self, column, column_index=None, result_format=None, include_config=False, 
             catch_exceptions=None, meta=None
     ):
 
         if column in self:
             return {
-                "success": (column_idx is None) or (self.columns.get_loc(column) == column_idx)
+                "success": (column_index is None) or (self.columns.get_loc(column) == column_index)
             }
 
         else:

--- a/tests/test_sets/expect_column_to_exist_test_set.json
+++ b/tests/test_sets/expect_column_to_exist_test_set.json
@@ -24,7 +24,7 @@
     "notes": "Positive test with column order",
     "in":{
       "column": "c2",
-      "column_idx": 1
+      "column_index": 1
     },
     "out":{
         "success": true
@@ -33,7 +33,7 @@
     "notes": "Column exists but wrong index",
     "in": {
       "column": "c3",
-      "column_idx": 4
+      "column_index": 4
     },
     "out":{
       "success": false 

